ols/customize/aap/reranker.py

```diff
@@ -0,0 +1,14 @@
+"""Reranker for post-processing the Vector DB search results."""
+
+import logging
+
+from llama_index.core.schema import NodeWithScore
+
+logger = logging.getLogger(__name__)
+
+
+def rerank(retrieved_nodes: list[NodeWithScore]) -> list[NodeWithScore]:
```

Comment:
@TamiTakamiya this https://github.com/ansible/ansible-chatbot-service/blob/main/ols/src/query_helpers/docs_summarizer.py#L13 seems to me a bug. It should probably
`import ols.customize.reranker` 
that way, the intended customize package's `reranker` will be used.  As of now, it will always be the `ols.customize.ols.reranker`

Reply:
nvm, I saw your fix

Reply:
we should probably have this as a default for all and so we won't need this here?